namespaces/live-1.cloud-platform.service.justice.gov.uk/claim-criminal-injuries-compensation-dev/resources/versions.tf

```diff
@@ -1,4 +1,3 @@
-
 terraform {
-  required_version = ">= 0.12"
-}
+  required_version = "0.12.17"
```

Comment:
I would leave the required version at ">=0.12" for now. The pipeline uses 0.12.30 and we're in the process of a migration (be it a little slow).